calculator.rb

```diff
@@ -0,0 +1,26 @@
+puts "Welcome to your calculator. Choose from the following operations: \n 1. add (+) \n 2. subtract (-) \n 3. multiply (*) \n 4. divide (/) \n Please type the name or symbol for the operator you would like to use"
+
+chosen_operator = gets.chomp!
+
+#gets numbers from user
+  puts "First number:"
+  first_number = gets.chomp.to_f
```

Comment:
You've written almost exactly the same code twice to get the two numbers from the user. As is it's only two lines, so it's not so bad, but it makes it difficult to expand that logic. For example, you might want to run this code in a loop until the user enters a valid number. Doing this in a method would mean you only have to write that code once.